be/src/runtime/lake_tablets_channel.cpp

```diff
@@ -75,7 +75,7 @@ class LakeTabletsChannel : public TabletsChannel {
 
     void abort() override;
 
-    void abort(const std::vector<int64_t>& tablet_ids) override { return abort(); }
+    void abort(const std::vector<int64_t>& tablet_ids, const std::string& reason) override { return abort(); }
```

Comment:
Why not use Status as the root cause?

Reply:
this function called by remote node through rpc, It is not only used for abort when there is an exception, but also for partitions whose data is empty which abort is normal.